bin/analyze_hoc_activity

```diff
@@ -100,7 +113,7 @@ def main()
     csedweek_students:        4875091,
     csedweek_countries:       356,
     petition_signatures:      PEGASUS_REPORTING_DB_READONLY[:forms].where(kind: 'Petition').count,
-    lines_of_code:            lines_of_code, #JSON.parse(Net::HTTP.get(URI('http://archive.learn.code.org/line_data.json')))['all'],
+    lines_of_code:            lines_of_code,
```

Comment:
My assumption is that, at some time in the past, the value was obtained through the above mechanism.